docs/mycoind/iovcore.rst

```diff
@@ -0,0 +1,48 @@
+---------------------
+Using IOV-Core Client
+---------------------
+
+While the blockchain code is in golang, we have developed a TypeScript (javascript-compatible) client side sdk
```

Comment:
The language [is called **Go**](https://tip.golang.org/doc/faq#go_or_golang). Maybe it is better to write "code is written in Go language" instead?  :nail_care:

Reply:
Sure thing